extensions/notebook/src/book/remoteBookController.ts

```diff
@@ -73,7 +73,7 @@ export class RemoteBookController {
 	public async getAssets(release?: IRelease): Promise<IAsset[]> {
 		if (release) {
 			let format: string[] = [];
-			if (utils.getOSPlatform() === utils.Platform.Windows || utils.getOSPlatform() === utils.Platform.Mac) {
+			if (process.platform === winPlatform || process.platform === macPlatform) {
```

Comment:
Do you need the constants elsewhere for something? If they're just being used to compare you can just use the string values - process.platform is a string literal union so the type checking will already guarantee you don't mistype it or something

Reply:
Looks like just for comparison to what process.platform returns